app/src/main/kotlin/at/bitfire/davdroid/ui/TasksActivity.kt

```diff
@@ -188,7 +191,13 @@ fun TasksCard(
 
     Scaffold(
         modifier = modifier,
-        snackbarHost = { SnackbarHost(snackbarHostState) }
+        snackbarHost = { SnackbarHost(snackbarHostState) },
+        topBar = {
+            (context as? Activity)?.BasicTopAppBar(
```

Comment:
This cast doesn't look stable / clean, especially if it's used more than one time.

I have used it myself for when an Activity is explicitly required, but in this case there's no need for an Activity. We can just pass an `onNavigateUp` argument around. (See comment in BasicTopAppBar)